src/quo2/components/inputs/search_input/view.cljs

```diff
@@ -0,0 +1,88 @@
+(ns quo2.components.inputs.search-input.view
+  (:require [oops.core :as oops]
+            [quo2.components.icon :as icon]
+            [quo2.components.inputs.search-input.style :as style]
+            [react-native.core :as rn]
+            [reagent.core :as reagent]))
+
+(def ^:private tag-separator [rn/view {:style style/tag-separator}])
+
+(defn- inner-tags
+  [tags-coll]
+  (into [rn/view {:style style/tag-container}]
+        (interpose tag-separator)
+        tags-coll))
+
+(defn- clear-button
+  [{:keys [on-press blur? override-theme]}]
+  [rn/touchable-opacity
+   {:style    style/clear-icon-container
+    :on-press on-press}
+   [icon/icon :i/clear
+    {:color (style/clear-icon blur? override-theme)
+     :size  20}]])
+
+(defn handle-backspace
+  [^js/Object event scroll-view-ref]
+  (when (= (oops/oget event "nativeEvent.key") "Backspace")
+    (.scrollToEnd scroll-view-ref #js {:animated false})))
```

Comment:
For good measure, it's usually better if we check the deref'ed `ref` is not nil before interacting with it. I've had a couple issues related to nil refs. It may be working now, but down the line the code may change and the ref may become nil due to the component being unmounted.

Reply:
Sounds good! :+1:  added